gocd/checkers.suse.gocd.yaml.erb

```diff
@@ -312,3 +312,31 @@ pipelines:
                 osc ci -m 'Fix repoid'
                 osc sr --yes SUSE:SLE-15-<%= servicepack %>:Update:QR -m 'Update Package List'
 <% end -%>
+<% ['SP3'].each do |servicepack| -%>
+  SLE15.<%= servicepack %>.QR.Setup:
+    group: SLE.Checkers
+    lock_behavior: unlockWhenFinished
+    environment_variables:
+      OSC_CONFIG: /home/go/config/oscrc-maintenance-installcheck
+    timer:
+      spec: 0 0 0 ? * THU
```

Comment:
I think a manual pipeline doesn't need a timer :)

Reply:
Fixed. Copied/pasted pipeline setup from SLE15.SP3.QR without double checking it :(